Deployments/suex-enginfprojects-test/k8s-manifests/deploy-mongodb.yml

```
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: enginfprojects-mongodb-master
+  namespace: suex-enginfprojects-test
+  labels:
+    app.kubernetes.io/name: enginfprojects-mongodb-master
+    app.kubernetes.io/component: database
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: enginfprojects-mongodb-master
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: enginfprojects-mongodb-master
+    spec:
+      containers:
+        - name: mongodb
+          image: mongo:4.4-bionic
```

```suggestion
          image: dockerhub.vshn.net/library/mongo:4.4-bionic
```
To workaround the rate-limit issue of Docker: https://www.docker.com/increase-rate-limit